docs/README.md

```diff
@@ -7,30 +7,29 @@ If you are a developer using Fn through the API, this section is for you.
 ### Getting Started
 
 * [Quickstart](https://github.com/fnproject/fn#quickstart)
-* [API Reference](http://petstore.swagger.io/?url=https://raw.githubusercontent.com/fnproject/fn/master/docs/swagger.yml)
+* [Function file (func.yaml)](developers/function-file.md)
+* [FDKs](developers/fdks.md)
+* [Config Vars for Functions](developers/configs.md)
+* [Hot functions](developers/hot-functions.md)
+* [Testing functions](developers/testing.md)
+* [Applications - groups of functions](developers/apps.md)
+* [Async functions](developers/async.md)
 * [FAQ](faq.md)
-* [Object Model](developers/model.md)
```

Comment:
intentional to remove object model?

Reply:
ya, don't think it's that useful and we don't keep it up to date. Can get same info from swagger.